app/src/test/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntryUiRendererTest.kt

```diff
@@ -58,4 +59,46 @@ class BloodSugarEntryUiRendererTest {
     // then
     verify(ui).showFastingBloodSugarTitle()
   }
+
+  @Test
+  fun `it should render the edit title when updating the random blood sugar`() {
+    // given
+    val testUserClock = TestUserClock()
+    val year = LocalDate.now(testUserClock).year
+    val bloodSugarEntryModel = BloodSugarEntryModel.create(year, Update(bloodSugarUuid, Random))
+
+    // when
+    bloodSugarEntryUiRenderer.render(bloodSugarEntryModel)
+
+    // then
+    verify(ui).showEditRadomBloodSugarTitle()
+  }
+
+  @Test
+  fun `it should render the edit title when updating the post prandial blood sugar`() {
+    // given
+    val testUserClock = TestUserClock()
+    val year = LocalDate.now(testUserClock).year
+    val bloodSugarEntryModel = BloodSugarEntryModel.create(year, Update(bloodSugarUuid, PostPrandial))
+
+    // when
+    bloodSugarEntryUiRenderer.render(bloodSugarEntryModel)
+
+    // then
+    verify(ui).showEditPostPrandialBloodSugarTitle()
+  }
+
+  @Test
+  fun `it should render the edit title when updating the fasting blood sugar`() {
+    // given
+    val testUserClock = TestUserClock()
+    val year = LocalDate.now(testUserClock).year
+    val bloodSugarEntryModel = BloodSugarEntryModel.create(year, Update(bloodSugarUuid, Fasting))
+
+    // when
+    bloodSugarEntryUiRenderer.render(bloodSugarEntryModel)
+
+    // then
+    verify(ui).showEditFastingBloodSugarTitle()
```

Comment:
Should we also add `verifyNoMoreInteractions(ui)` after the assertions in these methods?

Reply:
Yes, forgot about that